impl/src/main/java/com/okta/sdk/impl/client/DefaultClientBuilder.java

```diff
@@ -233,6 +233,10 @@ else if (SYSPROPS_TOKEN.equalsIgnoreCase(location)) {
             clientConfig.setPrivateKey(props.get(DEFAULT_CLIENT_PRIVATE_KEY_PROPERTY_NAME));
         }
 
+        if (Strings.hasText(props.get(DEFAULT_CLIENT_KID_PROPERTY_NAME))) {
+            clientConfig.setKid(props.get(DEFAULT_CLIENT_KID_PROPERTY_NAME));
+        }
```

Comment:
I think it would not be superfluous to add a test to `DefaultClientBuilderTest` with "kid" property in okta.yaml.